setup.py

```diff
@@ -20,7 +20,7 @@ def _get_version():
     long_description=open('README.md').read(),
     url="https://github.com/openshift/doozer",
     license="Apache License, Version 2.0",
-    packages=["doozerlib"],
+    packages=find_packages(),
```

Comment:
```suggestion
    packages=find_packages(exclude=("tests",)),
```

Reply:
so you don't need to get rid of `__init__.py` on test modules.

Reply:
Thanks for the hint. I did some tests and found `test.*` should also be specified in order to exclude subpackages.